app/src/pages/playground/PlaygroundExperimentRunDetailsDialog.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useMemo } from "react";
+import { graphql, useLazyLoadQuery } from "react-relay";
+import { Panel, PanelGroup, PanelResizeHandle } from "react-resizable-panels";
+import { css } from "@emotion/react";
+
+import {
+  Card,
+  CardProps,
+  Dialog,
+  Flex,
+  Heading,
+  Icon,
+  Icons,
+  Text,
+  View,
+} from "@arizeai/components";
+
+import { CopyToClipboardButton, ViewSummaryAside } from "@phoenix/components";
+import { JSONBlock } from "@phoenix/components/code";
+import { resizeHandleCSS } from "@phoenix/components/resize";
+import { LatencyText } from "@phoenix/components/trace/LatencyText";
+
+import type { PlaygroundExperimentRunDetailsDialogQuery } from "./__generated__/PlaygroundExperimentRunDetailsDialogQuery.graphql";
+
+/**
+ * A slide-over that shows the details of a playground experiment run.
+ */
+export function PlaygroundExperimentRunDetailsDialog({
+  runId,
+}: {
+  runId: string;
+}) {
+  const data = useLazyLoadQuery<PlaygroundExperimentRunDetailsDialogQuery>(
+    graphql`
+      query PlaygroundExperimentRunDetailsDialogQuery($runId: GlobalID!) {
+        run: node(id: $runId) {
+          ... on ExperimentRun {
+            output
+            startTime
+            endTime
+            error
+            example {
+              id
+              revision {
+                input
+                output
+              }
+            }
+            annotations {
+              edges {
+                annotation: node {
+                  id
+                  name
+                  label
+                  score
+                  explanation
+                  annotatorKind
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    { runId }
+  );
+  const run = data.run;
+  const exampleId = run.example?.id;
+  const revision = run.example?.revision;
+  const input = revision?.input;
+  const referenceOutput = revision?.output;
+  return (
+    <Dialog
+      title={`Experiment Run for Example: ${exampleId}`}
+      size="fullscreen"
+    >
+      <PanelGroup direction="vertical" autoSaveId="example-compare-panel-group">
+        <Panel defaultSize={100}>
```

Comment:
These sizes are a percentage of the total height of the PanelGroup component and should not exceed 100 nor represent pixels. You likely want something like `33` on  this panel and `66` on the other. 

https://github.com/bvaughn/react-resizable-panels/tree/main/packages/react-resizable-panels#panel